File: between-two-sets.py
def getTotalX(a, b):
    count = 0
    first = max(a)
    second = max(b)
    for i in range(first,second+1):
        div = True
        for num in a:
            if i % num !=0:
                div = False
        for num in b:
            if num % i !=0:
                div = False
        if div:
            count += 1
    return count

File: Java/between-two-sets.java
class Result {
    public static int getTotalX(List<Integer> a, List<Integer> b) {
        int count = 0;
        int first = Collections.max(a);
        int last = Collections.min(b);
        for(int i=first;i<=last;i++){
            boolean div = true;
            for(Integer num : a){
                if(i%num != 0){
                    div = false;
                }
            }
            for(Integer num : b){
                if(num % i != 0){
                    div = false;
                }
            }
            if(div){
                count++;
            }
        }
        return count;
    }
}